frontend/vue.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    // changed to put dist inside backend folder
+    outputDir: '../backend/dist',   
+
+    // relative to outputDir
+    // moves css, js, and img inside a static folder
+    assetsDir: 'static',     
+
+    // Need to restart when making changes here
+    // Side note: Wasted about two hours
+    devServer: {
+        proxy: {
+            '^/api': {
+                target: 'http://localhost:5000',
```

Comment:
Bei einem Pullrequest, der Auf dem Production Server landen soll, sollten keine Pfade mehr auf den Localhost zeigen. Dein Backend wir auf http://server.it-humke.de:9003 liegen. Das bitte vor einem PR immer ändern